mmv1/third_party/terraform/services/redis/resource_redis_cluster_test.go.tmpl

```diff
@@ -263,14 +262,78 @@ func TestAccRedisCluster_createUpdateDeletionProtection(t *testing.T) {
 			},
 			{
 				// update deletion protection to false and delete the cluster
-        		Config: createOrUpdateRedisCluster(&ClusterParams{name: name, replicaCount: 0, shardCount: 3, deletionProtectionEnabled: false, zoneDistributionMode: "MULTI_ZONE"}),
+				Config: createOrUpdateRedisCluster(&ClusterParams{name: name, replicaCount: 0, shardCount: 3, deletionProtectionEnabled: false, zoneDistributionMode: "MULTI_ZONE"}),
 			},
+		},
+	})
+}
+{{ end }}
+func TestAccRedisCluster_automatedBackupConfig(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
 
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckRedisClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccRedisCluster_automatedBackupConfig(context),
+			},
+			{
+				ResourceName:            "google_redis_cluster.cluster_abc",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"automated_backup_config", "automated_backup_config.0", "automated_backup_config.0.fixed_frequency_schedule", "automated_backup_config.0.fixed_frequency_schedule.0.start_time", "utomated_backup_config.0.fixed_frequency_schedule.0.start_time.0.hours", "automated_backup_config.0.retention"},
```

Comment:
ISVI here indicates an issue with import- there's no good reason we can't match both states exactly here. (The fix to not check state in Read should fix this)

Reply:
The API docs are still being worked on right now. but they the implmetation is the same as the ones I have provided :)

Reply:
Totally agree, should be good now :)